tox.ini

```diff
@@ -28,8 +33,8 @@ deps =
 
     # test with the latest builds of Django and django-rest-framework
     # to get early warning of compatibility issues
-    djmaster: https://github.com/django/django/archive/master.tar.gz
-    djmaster: https://github.com/ottoyiu/django-cors-headers/archive/master.tar.gz
+    djmaster: https://github.com/django/django/archive/main.tar.gz
+    djmaster: https://github.com/ottoyiu/django-cors-headers/archive/main.tar.gz
```

Comment:
Looks like it was added #415 to provide the dependency for each django version there. Somewhere along the line those were removed except its listing under djmaster.  I agree with you it is superfluous since the djmaster test is only diagnostic anyway.

Reply:
Removed in the next commit.